core/src/main/java/org/frankframework/management/bus/endpoints/SecurityItems.java

```diff
@@ -227,8 +227,8 @@ private ArrayList<Object> addSapSystems() {
 
 			sapSystems = (List) factoryGetRegisteredSapSystemsNamesAsList.invoke(sapSystemFactory, (Object[])null);
 			factoryGetSapSystemInfo = c.getMethod("getSapSystemInfo", String.class);
-		} catch (Exception e) {
-			log.debug("Caught NoClassDefFoundError, just no sapSystem available: " + e.getMessage());
+		} catch (Throwable t) { //Don't change this throwable, AI doesn't know better...
```

Comment:
Instead of blaming AI, I think a `@SuppressWarning` would be better, because static code analysis tools will complain about it, for sure.